dev/src/index.ts

```diff
@@ -279,11 +279,6 @@ export class Firestore {
    * use `Timestamp` now and opt-in to this new behavior as soon as you can.
    */
   constructor(settings?: Settings) {
```

Comment:
can you please update the typing information for Settings?

Reply:
I don't want to expose this setting in the public API. Hence, it is missing from both the `.d.ts` file and the JSDoc documentation. Note that they way the settings are defined allows you to pass any arbitrary key-value pair.